Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamPictureInPictureController.swift

```diff
@@ -20,29 +20,29 @@ final class StreamPictureInPictureController: NSObject, AVPictureInPictureContro
 
     /// The RTCVideoTrack for which the picture-in-picture session is created.
     @MainActor
-    public var track: RTCVideoTrack? {
+    var track: RTCVideoTrack? {
```

Comment:
the class wasn't public at all, right?

Reply:
That's right. The class wasn't public but it had properties and methods that were public. Here i'm just removing the unnecessary modifiers